### PR TITLE
Server

### PR DESCRIPTION
**Pull request by Cofucan**
Code Refactoring, PEP8 Compliance
​
## Description
* Changed the password of anonymous user to a more appropriate and secure password that is definitely not an Easter egg
* Fixed a multitude of typos introduced by someone who must not be named
* Removed unused imports
* Applied PEP8 standards and added type hints
* Updated docstrings
* Applied validation to user-provided directory names for security purposes
​
## Related Issue (Link to linear ticket)
None specifically
​
## Motivation and Context
To make code to follow the style of the project
​
## How Has This Been Tested?
Most of the changes are documentation changes

## Screenshots (if appropriate - Postman, etc):
​
## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
​
## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: app/routes/auth_routes.py [hunk 1]
```diff
@@ -131,18 +126,15 @@ async def login_user(
 
     actual_user_password = needed_user.hashed_password
 
-    # Validating the entered password
-    result = bcrypt.checkpw(hashed_password, actual_user_password)
-
-    if not result:
+    if _ := bcrypt.checkpw(hashed_password, actual_user_password):
```
Comment (DestinedCodes) [inline app/routes/auth_routes.py:129]: Writ this to be mire readable 9kease if actual password is bcrypy(), return:

File: app/models/user_models.py [hunk 2]
```diff
@@ -16,14 +16,14 @@
...
-    """ The user model """
+    """The user model"""
+
     __tablename__ = "users"
 
     id: int = Column(Integer, primary_key=True, index=True)
     username: str = Column(
         String, index=True, unique=True, nullable=False, default=None
     )
-    email: str = Column(String, nullable=True, default=None)
```
Comment (DestinedCodes) [inline app/models/user_models.py:26]: Why did you delete the email field from the user midek

Comment (Cofucan) [reply app/models/user_models.py:26]: The email field is still there, it was a duplicate

File: app/routes/auth_routes.py (hunk 1 above)
Comment (Cofucan) [reply app/routes/auth_routes.py:129]: It was a Sourcery suggestion
